site/content/en/docs/manual/advanced/annotation-with-brushing-tools.md

```diff
@@ -0,0 +1,121 @@
+
+You can use annotation with brush together with polygon-to-mask annotation on the same image.
+
+See:
+
+- [Brushing tools menu](#brushing-tools-menu)
+- [Annotation with brush](#annotation-with-brush)
+- [Annotation with polygon-to-mask](#annotation-with-polygon-to-mask)
+- [Remove underlying pixels](#remove-underlying-pixels)
+- [Import and export](#import-and-export)
+
+## Brushing tools menu
+
+The brushing tool menu appears on the top of the screen after you click **Draw a mask**:
+
+![BT Menu](/images/brushing_tool_menu.png)
+
+It has the following elements:
+
+|Element|Description|
+|----|-----|
+|![Tick icon](/images/tick_icon.png)| **Save mask** saves the created  mask. The saved mask will appear on the object sidebar|
+|![Add label](/images/brushing_tools_add_label.png)| **Add label** adds a new label to the object sidebar, brushing tool menu switches to **Brush**.|
+|![Brush](/images/brushing_tools_icon.png)| **Brush** applies the mask to the object on the image.|
+|![Eraser](/images/brushing_tools_erase.png)|**Eraser** removes part of the mask from the image.|
+|![Add poly](/images/brushing_tools_add_poly.png)|**Polygon**  selection tool. Selection will be converted to the mask.|
+|![Remove poly](/images/brushing_tools_remove_poly.png)|**Remove polygon selection** removes a part of the polygon selection.|
+|![Brush size](/images/brushing_tools_brush_size.png)|**Brush size** in pixels. <br>**Note:** Visible only when **Brush** is selected.|
+|![Brush shape](/images/brushing_tools_brush_shape.png)|**Brush shape** with two options: circle and square. <br>**Note:** Visible only when **Brush** is selected.|
+|![Pixel remove](/images/brushing_tools_pixels.png)|**Remove underlying pixels**. When you are drawing or editing a mask with this tool, <br>pixels on other masks that are located at the same positions as the pixels of the <br>current mask are deleted.|
+|![Label](/images/brushing_tools_label_drop.png)|**Label** that will be assigned to the  mask|
+|![Move](/images/brushing_tools_brush_move.png)|**Move**. Click and hold to move the menu bar to the other place on the screen|
+
+
+
+## Annotation with brush
+
+Prerequisites: [Create task](/docs/manual/basics/creating_an_annotation_task/), upload the image, add labels, and open the workspace.
+
+Do the following:
+
+1. From the [controls sidebar](/docs/manual/basics/controls-sidebar/), select **Brush** ![Brush icon](/images/brushing_tools_icon.png).
+2. In the **Draw new mask** menu, select label for your mask, and click **Draw a mask**. <br>The **Brush**![Brush](/images/brushing_tools_icon.png) tool will be selected by default.
+
+   ![BT context menu](/images/brushing_tools_context_menu.png)
+
+3. With the brush, draw a mask on the object you want to label. <br>To correct selection, use **Eraser** ![Eraser](/images/brushing_tools_erase.png)
+
+   ![Brushing](/images/brushing_tools.gif)
```

Comment:
Just in my opinion, I would suggest to use more contrast colors to demonstrate the tool usage. Here we see brown on brown and it's difficult to recognize the annotation result.

Reply:
updated